utilities/getSIMetricPrefixData.ts

```diff
@@ -0,0 +1,27 @@
+export default function getSIMetricPrefixData(num: number): string {
```

Comment:
Keep in mind: do not use "data" for identifiers. "data" is unclear. it means nothing. So `getSIMetricPrefixData` means the same with `getSIMetricPrefix`. If you wanna intend something, define it clearly. Probably `String`, `DataString` or things like that. You will **never** face with appropriate situation to use `xxxData` in your developer life.

Reply:
By the way, it should be "suffix" yet "prefix", I guess.